etc/firejail-default

```diff
@@ -30,7 +30,8 @@ profile firejail-default flags=(attach_disconnected,mediate_deleted) {
 /{,var/}run/user/**/dconf/user rw,
 /{,var/}run/user/**/pulse/ rw,
 /{,var/}run/user/**/pulse/** rw,
-/{,var/}run/user/**/*.slave-socket rw,
+/{,var/}run/user/**/*.slave-socket rwl,
+/{,var/}run/user/**/#@{PID} rw,
```

Comment:
I'm not that familiar with apparmor, but since it appears that any PID can be substituted here, would this allow any program to have rw privs in /run/user or /var/run/user?  
If so, does that have any security implications?

Maybe I'm completely off-base here... :smile:

Reply:
It isn't related to "Process ID", it just catches string of random numbers, see example denial messages I posted in description. I used existing alias https://github.com/netblue30/firejail/blob/master/etc/firejail-default#L11 because it fits for the job.